src/v/pandaproxy/api/api-doc/schema_registry.json

```diff
@@ -862,4 +862,38 @@
           }
         }
       }
+    },
+    "/admin/redpanda-services/restart": {
+      "put": {
+        "summary": "Restart a redpanda service.",
+        "operationId": "put_admin_redpanda_services_restart",
+        "consumes": [
+          "application/vnd.schemaregistry.v1+json",
+          "application/vnd.schemaregistry+json",
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "service",
+            "in": "query",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "produces": ["application/vnd.schemaregistry.v1+json"],
+        "responses": {
+          "204": {
+            "description": "Restart success",
+            "schema": {
+              "type": "object"
+            }
+          },
+          "500": {
+            "description": "Internal Server error",
+            "schema": {
+              "$ref": "#/definitions/error_body"
+            }
+          }
```

Comment:
What about 404 if the service doesn't exist?

Reply:
Good catch, I missed adding 404 in when I also added in 404 on the C++ side.